server/src/http_history.go

```diff
@@ -63,5 +63,5 @@ func httpHistory(c *gin.Context) {
 	if len(playerNames) > 1 {
 		data.NamesTitle = "Game History for [" + strings.Join(playerNames, ", ") + "]"
 	}
-	httpServeTemplate(w, data, "profile", "history")
+	httpServeTemplate(w, data, "profile-new", "history")
```

Comment:
why are we using profile-new instead of profile

Reply:
I was going back and forth between profile and profile-new. In the end I managed to make all the pages as I liked, but I forgot to transfer back.